samples/sensor/icp101xx/README.rst

```diff
@@ -0,0 +1,51 @@
+.. _icp101xx:
+
+ICP101xx: Barometric Pressure and Temperature Sensor
+############################################
+
+Description
+***********
+
+This sample application periodically measures the sensor
+temperature and pressure, displaying the 
+values on the console along with a timestamp since startup.
+It also displays the estimated altitude.
+
+Wiring
+*******
+
+This sample uses an external breakout for the sensor.  A devicetree
+overlay must be provided to identify the SPI bus and GPIO used to
```

Comment:
There is no GPIO to control the sensor? And the communication is I2C !

Reply:
Fixed ! Thanks